axelor-budget/src/main/resources/views/GlobalBudget.xml

```diff
@@ -79,7 +79,7 @@
       <panel name="globalBudgetLevelButtonPanel" colSpan="3" stacked="true">
         <button name="draftBtn" title="draftBtn"
           showIf="(statusSelect == 1 || statusSelect == 2) &amp;&amp; totalAmountCommitted == 0 &amp;&amp; realizedWithNoPo == 0 &amp;&amp; realizedWithPo == 0"
-          colSpan="12" onClick="action-global-budget-method-draft-children, save"/>
+          colSpan="12" onClick="action-global-budget-group-draft-children, save"/>
```

Comment:
if you use `setReload(true)` in the Controller, the `save` should be before the call to the method, and not after, so please change it.